styles/main.css

```diff
@@ -24,25 +29,74 @@ body {
   bottom: 0;
   position: absolute;
   height: 3px;
-  width: 200px;
+  width: 150px;
   left: 50%;
   transform: translateX(-50%);
 }
 
-.wrapper {
+.wrapper-about {
   display: grid;
   grid-template-columns: repeat(2, auto);
   grid-auto-rows: minmax(100px, auto);
   grid-gap: 10px;
   justify-content: center;
 }
 
+.what-is, .how-it-works {
+  min-height: 40vh;
+}
+
 .what-is p {
   margin: 1rem 0;
   padding: 0 2rem;
 }
 
+.wrapper-features {
+  display: grid;
+  grid-template-columns: repeat(3, 392px);
+  grid-auto-rows: minmax(100px, auto);
+  justify-content: center;
+}
+
+.how-it-works {
+  background-color: #c74939;
+  padding: 1rem;
+}
+
+.card {
+  align-items: center;
+  background-color: #40596b;
+  border-radius: 5px;
+  margin: 1rem;
+  position: relative;
+  min-height: 30rem;
+}
```

Comment:
I thought adding a shadow effect around each of the card divs might look cool. Here is how I did it in Chrome Dev Tools:

`box-shadow: 5px 4px 3px rgba(64,89,107, 0.5), 0px 8px 13px rgba(0,0,0,0.4), 0px 18px 23px rgba(0,0,0,0.4);`

This is just cosmetic and I know not everyone likes this kind of shadow effect, so if you like the clean look better that's totally okay with me!